<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreakWindow</class>
 <widget class="QMainWindow" name="BreakWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>~Experiment~</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{background: white;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>9</y>
      <width>1210</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="5,0,1">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="leftMargin">
      <number>200</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QTextEdit" name="instructionsBox">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>AvenirNext LT Pro Regular</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit{border: 1px solild gray;}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="timeLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>AvenirNext LT Pro Regular</family>
         <pointsize>17</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit{border: 1px solild gray;}</string>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>AvenirNext LT Pro Regular</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
